moped-editor/src/auth/user.js

```diff
@@ -109,7 +109,6 @@ export const initializeUserDBObject = (userObject) => {
     }).then((res) => {
       // Then we parse the response
       res.json().then((resData) => {
-        // If we have any user data,  use it
```

Comment:
i'm v confused about the commit history in this PR. my local `git log` looks normal 🤷‍♂️ 

i will try rebasing after y'all review.

Reply:
definitely a puzzler! Ignored ✅